src/components/achievements/Achievements.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import CardHeader from '../cardheader/CardHeader';
+import EditOptions from '../editoptions/EditOptions';
+import { Add } from '../../assets/image';
+import CardFooter from '../cardfooter/CardFooter';
```

Comment:
Instead of these, you can use root imports as follows:
```suggestion
import CardHeader from 'components/cardheader/CardHeader';
import EditOptions from 'components/editoptions/EditOptions';
import { Add } from 'assets/image';
import CardFooter from 'components/cardfooter/CardFooter';
```

You can do these by adding `jsconfig.json` file in the root folder, and add the following contents in the file:
{
  "compilerOptions": {
    "target": "ES6",
    "moduleResolution": "node",
    "baseUrl": "./src"
  },
  "exclude": [
    "node_modules",
    "lib",
    ".tmp",
    "build"
  ]
}

Reply:
🆗